### PR TITLE
SC 1.3.4 Thorough Review

### PR DESCRIPTION
**Pull request by nitedog**
Addressing issues from [thorough review](https://www.w3.org/2002/09/wbs/1/WCAGvideos_batch1thorough/results#xq8):
- Removed scenes with table stand, which caused confusion
- Removed reference to *news* app, to avoid confusion

### CODE REVIEW

**Review by jade-mc — APPROVED** [review]

File: content/SC-1.3.4.md [hunk 1]
```diff
@@ -53,9 +53,7 @@ This page provides the script for Success Criterion 1.3.4 Orientation of the [WC
...
-| 6 | @:@@ - @:@@ | Fortunately his favorite newspaper app works well in both orientations, and does not require him to use any particular one. | We see Jan switching to another newspaper app that works well in landscape orientation while Jan is cruising away. |
+| 1 | 0:00 - 0:@@ | Jan has cerebral palsy, which, in his case, means limited use of his arms. | Jan is using a wheelchair (power chair) with a headrest and seatbelt to support him from the involuntary movement of his arms and legs. |
+| 2 | @:@@ - @:@@ | Jan uses a tablet mounted on his wheelchair, where he cannot change its orientation. | We zoom in on the tablet mounted to his wheelchair. |
+| 3 | @:@@ - @:@@ | Unfortunately some apps only work in portrait orientation, so Jan can not read the articles when the tablet is mounted on his wheelchair. | We see the screen mounted in landscape orientation with the text in an app appearing in portrait (sideways for him). |
```
Comment (bakkenb) [inline content/SC-1.3.4.md]: Will we have trouble with this scene from an audio described perspective? Seems like the audio is not giving enough of the visual issue of the tablet being mounted in landscape mode only. Someone only listening to this video may not fully understand the issue of the orientation from this example without some audio description.

Maybe add, "... in landscape orientation." to the end of this sentence???

Comment (nitedog) [reply content/SC-1.3.4.md]: Add visual of  'door slam' message like "Please turn your device to portrait orientation" or similar. (To address comments from Detlev and Rain)

File: content/SC-1.3.4.md [hunk 2]
```diff
@@ -53,9 +53,7 @@ This page provides the script for Success Criterion 1.3.4 Orientation of the [WC
...
+| 1 | 0:00 - 0:@@ | Jan has cerebral palsy, which, in his case, means limited use of his arms. | Jan is using a wheelchair (power chair) with a headrest and seatbelt to support him from the involuntary movement of his arms and legs. |
+| 2 | @:@@ - @:@@ | Jan uses a tablet mounted on his wheelchair. In his case it is fixed to landscape orientation and he cannot change this orientation. | We zoom in on the tablet mounted to his wheelchair. |
+| 3 | @:@@ - @:@@ | Unfortunately some apps only work in portrait orientation, so Jan can not read the articles when the tablet is mounted on his wheelchair. | We see the screen mounted in landscape orientation with the text in an app appearing in portrait (sideways for him). |
+| 4 | @:@@ - @:@@ | Apps work well for Jan when they do not require him to use any particular orientation. Such apps work for him while he is using the wheelchair and elsewhere. | We see Jan switching to another app that works well in landscape orientation. |
```
Comment (nitedog) [inline content/SC-1.3.4.md]: Remove "and elsewhere" to address comment from Kim.

File: content/SC-1.3.4.md (hunk 1 above)
Comment (nitedog) [reply content/SC-1.3.4.md]: Sub-group suggests: add "... in landscape orientation" to the audio, and do not add discussion on annoying messages. Make sure the text appearing in portrait (sideways) is really difficult to read, to drive home the point.

File: content/SC-1.3.4.md (hunk 2 above)
Comment (nitedog) [reply content/SC-1.3.4.md]: Sub-group suggests: remove the entire sentence "Such apps work for him while he is using the wheelchair and elsewhere."

**Pull request merged**